content/07.pathogenesis.md

```diff
@@ -55,7 +55,8 @@ When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the popu
 R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
 Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1126/science.272.5258.74].
 
-R~0~ for COVID-19 is estimated to lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221]; estimates vary considerably depending on the data and the methods used.
+Estimates of R~0~ for COVID-19 lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221].
+Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations at specific times, therefore the different estimates of R~0~ should not necessarily be interpreted as a range of estimates of the same underlying parameter.
```

Comment:
```suggestion
Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations at specific times; therefore, the different estimates of R~0~ should not be interpreted as a range of estimates of the same underlying parameter.
```

Reply:
```suggestion
Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations in specific environments. The different estimates of R~0~ should not necessarily be interpreted as a range of estimates of the same underlying parameter.
```

Reply:
ok just changed ref to time since R0 should - by definition - be time-independent